recipe/meta.yaml

```
@@ -21,11 +21,13 @@ requirements:
   host:
     - python
     - python-build
+    - ninja-base  # [win]
     - meson-python >=0.13.1
     - meson >=1.2.1,<2
     - cython >=3.0.5,<4
-    - numpy {{ numpy}}
-    - versioneer
+    - numpy {{ numpy }}
+    - versioneer 0.29
+    - tomli 2.2.1
```

```suggestion
    - tomli  # [py<311]
```
See the expansion of `versioneer[toml]` https://github.com/python-versioneer/python-versioneer/blob/0.29/pyproject.toml#L37